app/assets/stylesheets/layouts/_col_marginless.scss

```diff
@@ -0,0 +1,42 @@
+.l-2col-even--marginless {
```

Comment:
That's a good idea. Does that signal to you the fact of no margin on the outer edges of the outer columns but continues to have gutter?